homework_2/task1.js

```diff
@@ -0,0 +1,17 @@
+const copyString = (str, n) => {
+
+    if (
+        typeof n === "number" &&
+        typeof str === "string" &&
+        n >= 0
+        ) {
+        return str.repeat(n);
+    } 
+
+    else return "Your number is less than 0";
```

Comment:
Don`t need "else" here and change the message to "Your number is less than 1".
Change this line to:
return "Your number is less than 1";